<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>VoiceFileWidget</class>
  <widget class="QWidget" name="VoiceFileWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>452</width>
        <height>117</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Selecting Voice File</string>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item row="0" column="0" >
        <widget class="QLabel" name="voiceFileLabel" >
          <property name="text" >
            <string>Path of the voice file:</string>
          </property>
        </widget>
      </item>
      <item row="0" column="1" >
        <widget class="KUrlRequester" name="voiceFileURL" >
          <property name="frameShape" >
            <enum>NoFrame</enum>
          </property>
          <property name="frameShadow" >
            <enum>Plain</enum>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="1" column="0" colspan="2" >
        <widget class="QGroupBox" name="genderOption" >
          <property name="title" >
            <string>Gender</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item row="0" column="0" >
              <widget class="QRadioButton" name="femaleOption" >
                <property name="text" >
                  <string comment="Female, as option for gender">Female</string>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="QRadioButton" name="maleOption" >
                <property name="text" >
                  <string comment="Male, as option for gender">Male</string>
                </property>
              </widget>
            </item>
            <item row="0" column="2" >
              <widget class="KPushButton" name="genderButton" >
                <property name="text" >
                  <string>Try to Determine From Voice File</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
